### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by skeeey**
Signed-off-by: liuwei <liuweixa@redhat.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -45,18 +45,31 @@ Repo maintainers can assign you an issue or pull request by leaving a
...
+  ```
+- run the unit test
+  ```shell
+  make test
+  ```
+- run the integration test
+  ```shell
+  make test-integration
+  ```
+- run the end-to-end test, prepare a kind cluster and run the following command
```
Comment (qiujian16) [inline CONTRIBUTING.md:60]: I think we need to load the image to kind at first?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
-place them under management.
+Contains controllers that support:
 
-## Community, discussion, contribution, and support
+- the registration of managed clusters to a hub to place them under management
+- the concept of clusterset (see [KEP-1645](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api))
+  by `ManagedClusterSet` API to group managed clusters
+- the concept of clusterclaim (see [KEP-2149](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid))
+  by `ManagedClusterClaim` API to collect the cluster information from a managed cluster
+- the management of [managed cluster add-ons](https://github.com/open-cluster-management-io/api/blob/main/addon/v1alpha1/types_managedclusteraddon.go)
```
Comment (qiujian16) [inline README.md:13]: also the link to the design doc here

File: README.md [hunk 3]
```diff
@@ -1,9 +1,15 @@
 # Cluster Registration
 
-Contains controllers that support the registration of managed clusters to a hub to
-place them under management.
+Contains controllers that support:
 
-## Community, discussion, contribution, and support
+- the registration of managed clusters to a hub to place them under management
+- the concept of clusterset (see [KEP-1645](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/1645-multi-cluster-services-api))
```
Comment (qiujian16) [inline README.md]: see .... for details

File: CONTRIBUTING.md (hunk 1 above)
Comment (skeeey) [reply CONTRIBUTING.md:60]: yes, updated

File: README.md (hunk 2 above)
Comment (skeeey) [reply README.md:13]: updated

File: README.md (hunk 3 above)
Comment (skeeey) [reply README.md]: updated

**Pull request merged**